src/content/docs/apis/nerdgraph/examples/nerdgraph-synthetics-tutorial.mdx

```diff
@@ -0,0 +1,745 @@
+        customHeaders: {
+          name: <var>HEADER_NAME</var>, 
+          value: <var>HEADER_VALUE</var>
+        }, 
+        redirectIsFailure: <var>REDIRECT_IS_FAILURE</var>, 
+        responseValidationText: <var>VALIDATION_TEXT</var>, 
+        shouldBypassHeadRequest: <var>BYPASS_HEAD_REQUEST</var>, 
+        useTlsValidation: <var>TLS_VALIDATION</var>
+      }
+    }
+  ) {
+    errors {
+      description
+      type
+    }
+  }
+}
+```
+
+  </Collapser>
+
+  <Collapser
+    className="freq-link"
+    id="update-simple-browser"
+    title="Update a simple browser monitor"
+  >
+```
+SOMETHING
+```
+  </Collapser>
+
+  <Collapser
+    className="freq-link"
+    id="update-scripted-browser"
+    title="Update a scripted browser monitor"
+  >
+```
+SOMETHING
+```
+  </Collapser>
+
+  <Collapser
+    className="freq-link"
+    id="update-API-test"
+    title="Update an API test monitor"
+  >
+```
+SOMETHING
+```
+  </Collapser>
+
+  <Collapser
+    className="freq-link"
+    id="update-step"
+    title="Update a step monitor"
+  >
+```
+SOMETHING
+```
+  </Collapser>
+
+  <Collapser
+    className="freq-link"
+    id="update-certificate-check"
+    title="Update a certificate check monitor"
+  >
+```
+SOMETHING
+```
+  </Collapser>
+
+  <Collapser
+    className="freq-link"
+    id="update-broken-links"
+    title="Update a broken links monitor"
+  >
+```
+SOMETHING
+```
+  </Collapser>
+</CollapserGroup>
+
+## Delete your synthetic monitors 
+
+Sometimes a monitor doesn't serve your needs anymore. Use these scripts to delete your monitors: 
+
+<CollapserGroup>
+  <Collapser
+    className="freq-link"
+    id=""
+    title="Delete monitors"
+  >
+
+  ```
+  mutation {
+    syntheticsDeleteMonitor (
+      guid: <var>ENTITY_GUID</var>
+      ) {
+      errors {
+        description
+      }
+    }
+  }
+  ```
+  </Collapser>
+  <Collapser
+    className="freq-link"
+    id=""
+    title="Delete scripted API monitors"
+  >
+
+  ```
+  SOMETHING
+  ```
+
+  </Collapser>
+</CollapserGroup>
+
+## Manage your private locations
+
+A [private location](/docs/synthetics/synthetic-monitoring/private-locations/priv[…]locations-overview-monitor-internal-sites-add-new-locations/) allows you to monitor applications behind your firewall. When you create a private location, you install and configure private minions to execute the monitors assigned to that private location. 
+
+<CollapserGroup>
+  <Collapser
+    className="freq-link"
+    id="private-location"
+    title="Create a private location"
+  >
+
+  ```
+  mutation{
+    syntheticsCreatePrivateLocation (
+      accountId: <var>NR_ACCOUNT_ID</var>,
+      name: "PrivateLocationName",
+      description: "Optional description",
+      verifiedScriptExecution: false
+    ) {
+      guid
+      errors {
+        description
+        type
+      }
+    }
+  }
+  ```
+
+  </Collapser>
+  <Collapser
+    className="freq-link"
+    id="private-location"
+    title="Update a private location"
+  >
+  
```

Comment:
just a note that some of these lack intro sentences. it might seem a bit redundant with the titles but I think it's good practice as it feels a bit abrupt to go straight into the example.